web-app/js/portal/filter/DateFilter.js

```diff
@@ -123,17 +122,14 @@ Portal.filter.DateFilter = Ext.extend(Portal.filter.Filter, {
     _getDateString: function(newDate) {
 
         if (newDate) {
-            newDate.setHours(this.timezoneCorrection);
```

Comment:
From memory (didn't use `git blame`) @dnahodil added this and he had a pretty solid reason. Simply removing that might introduce the problem he was trying to fix at the time.